sim_benchmark/README.md

```diff
@@ -0,0 +1,48 @@
+# Sim Benchmark
+
+A simple benchmark for pygazebo and pybullet simulator.
+
+
+The scene is a turtlebot with 2 randomly moving wheels and a camera sensor. 
+
+Pybullet incompatible sensors and plugins were removed from the turtlebot urdf file. The model file contains 2 continuous joints and 30 fixed joints.
+
+# Test Result  ( Core i9-7960X@2.8Ghz, RTX 2080 Ti)
+Note there are 100 substeps per one step
+
+Turtlebot pyBullet without camera sensor:
+11.5 steps/second
+
+Turtlebot pyBullet with camera sensor (image rendered by OpenGL) :
+15 steps/second
+
+Turtlebot pyBullet with camera sensor (image rendered by CPU) :
+14.50 steps/second
+
+
+Turtlebot pyGazebo without camera sensor:
+175 steps/second
+
+Turtlebot pyGazebo with camera sensor:
+98 steps/second
```

Comment:
We need to figure out why there is such big gap between pybullet and pygazebo, otherwise this benchmark is not convincing.

Reply:
I'm working on this problem.

Reply:
It seems that gazebo ignore some dynamics for fixed joints/links. Now I removed the fixed upper two layers for less fixed joints. Only the base/battery pack and wheels left. Now performance of pybullet without camera is better than pygazebo now. Case with camera still need checking. Those removed links will be restored when we can find a way that pygazebo and pybullet do the same thing with fixed joint.